### DISCUSSION TITLE
1.19+ Version

### DISCUSSION THREAD

**Discussion by TyffAx** [MrTJP/ProjectRed#1765] [Q&A] [answered]
Hello. The mod will be available for Minecraft 1.19.2 or above ? If yes, when approximately.

**Comment by MrTJP** [comment, marked as answer]
Yes. Soon

**Reply by TyffAx** [reply]
Sure, thank you.

**Reply by DJj123dj** [reply]
Oh thanks, It's one of my favorite mods, but I really want to use it in combination with the create mod in 1.19.2! I hope you update it in the coming week(s)!

**Reply by Markus90MG** [reply]
is there any new information about Projekt Red for 1.19.2?

**Reply by Superlasek** [reply]
Project Red its one of my fav mods, using it since 1.7.10 (I know that they are different authors), I hope soon an update for 1.19.2, you don't know how much we need it and have our redstone systems much better<3

**Reply by JKL213** [reply]
Yeah can't wait as well.

**Reply by Sa1ZeR** [reply]
1.19.2 pls :)

**Reply by schloops1** [reply]
Great... love Project Red. Thx a lot!
